sdk/contentsafety/azure-ai-contentsafety/tests/recordings/test_content_safety.pyTestContentSafetyCasetest_analyze_text.json

```diff
@@ -0,0 +1,55 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "00000000000000000000000000000000/contentsafety/text:analyze?api-version=2023-04-30-preview",
```

Comment:
This looks strange to me, left some comments to clean up the sanitizers.

Reply:
Fixed.